saleor/plugins/openid_connect/utils.py

```diff
@@ -397,8 +399,11 @@ def get_or_create_user_from_payload(
         "private_metadata": {oidc_metadata_key: sub},
         "password": make_password(None),
     }
+    cache_key = oidc_metadata_key + "-" + str(sub)
     try:
-        user = User.objects.get(**get_kwargs)
+        user = cache.get(cache_key)
```

Comment:
I don't think this belongs under `try` as `cache.get` can't raise `User.DoesNotExist`.